SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# Security Policy
+
+In **Amplica Labs** we take the security of our open-source software seriously. We
```

Comment:
Should this be Amplica labs or something else?

Reply:
`At` Amplica Labs

Reply:
We should likely just generalize it, or simplify.

```suggestion
We take the security of open-source software seriously. We
```

Dropping the intro line?
```suggestion
We
```

Like to avoid the we if possible.

Reply:
OK I'll remove the first line, open to suggestions what should I replace `We` with?

Reply:
I guess passive voice would be the other option. "The efforts of ... in helping maintain.. for for the users... is appreciated."

The "we" works good enough as the it refers to the "maintainers"